src/crimson/os/seastore/extentmap_manager/btree/extentmap_btree_node.h

```diff
@@ -0,0 +1,99 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+
+#pragma once
+
+#include "crimson/common/log.h"
+#include "crimson/os/seastore/seastore_types.h"
+#include "crimson/os/seastore/transaction_manager.h"
+#include "crimson/os/seastore/extentmap_manager.h"
+
+namespace crimson::os::seastore::extentmap_manager{
+
+struct ExtMapNode : LogicalCachedExtent {
+  TransactionManager* tm = nullptr;
```

Comment:
This isn't wise, just pass it through the methods along with Transaction.  Consider creating a struct combining just the TransactionManager& and the Transaction&.